CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
+## 2.7.0 (July 1, 2022)
+
+Minimum required Control Plane version: `v2.32.0`.
+Resource incompatible with Control Planes between `v2.29` and `v2.31`: parameter `certificate_bundle_secrets` from resource `cyral_sidecar`.
+
+### Bug fixes:
+
+- **Fix memory referencing issue and refactor resources**: [#223](https://github.com/cyralinc/terraform-provider-cyral/pull/223).
+
+### Features:
+
+- **Add resources for per-repository data map**: [#230](https://github.com/cyralinc/terraform-provider-cyral/pull/230).
+- **Add data source for cyral_integration_idp**: [#239](https://github.com/cyralinc/terraform-provider-cyral/pull/239).
+- **Support MongoDB repository replica sets**: [#228](https://github.com/cyralinc/terraform-provider-cyral/pull/228).
+
```

Comment:
I think its worth mentioning that we updated our docs with some new guides as well.
```suggestion

### Documentation:

- **Add Terraform guides for basic usages of Cyral features**: [#238](https://github.com/cyralinc/terraform-provider-cyral/pull/238).
```